app/assets/stylesheets/application.scss

```diff
@@ -23,3 +22,10 @@ $govuk-compatibility-govukelements: true;
 @import "modules/related-content";
 @import "modules/service-standard-point";
 @import "modules/typography";
+
+.sticky-element {
```

Comment:
Can we prefix this with `app-` and move it as a partial within what we call in this app `stylesheets/modules`?

Reply:
Yes! I thought of moving this somewhere after I created the PR, but wasn't sure where a good place would be so decided to wait for suggestions 😁

Reply:
Ideally, we'd have an ITCSS-based structure of folders, but this way at least we're making it consistent with the current approach and ease the transition – whenever that will happen.

Reply:
Thanks, changed.